src/modules/chat/index.js

```diff
@@ -54,24 +57,23 @@ function hasNonASCII(message) {
 class ChatModule {
     constructor() {
         watcher.on('chat.message', ($element, message) => this.messageParser($element, message));
-        watcher.on('channel.updated', ({bots}) => {
+        watcher.on('channel.updated', ({ bots }) => {
```

Comment:
undo all these cosmetic formatting changes? They're not relevant to this PR?

Reply:
completed.

Reply:
I also meant all the other extra whitespace added.

Reply:
Removed all white space, and newlines not relevant to code change.